src/govuk/components/input/index.js

```diff
@@ -41,10 +43,12 @@ function Input(props) {
       {errorMessageComponent}
       <input
         id={id}
+        autoFocus={autofocus}
```

Comment:
This could have been added already without the need for an explicit prop could it not? Since any additional props are received as `...attributes` and then spread into the underlying element

Reply:
@andymantell true. I will remove this.